tcms/testruns/static/testruns/js/get.js

```diff
@@ -33,8 +33,41 @@ $(document).ready(() => {
         }
     });
 
-    $('.add-hyperlink-bulk').click(function() {
+    $('.add-comment-bulk').click(() => {
+        const selected = selectedCheckboxes()
+        if ($.isEmptyObject(selected)) {
+            return false
+        }
+
+        const enterCommentText = $('#test_run_pk').data('trans-comment')
+        const comment = prompt(enterCommentText);
+        if (!comment) {
+            return false
+        }
+
+        selected.executionIds.forEach(executionId => {
+            jsonRPC('TestExecution.add_comment', [executionId, comment], comment => {
+                const testExecutionRow = $(`.test-execution-${executionId}`)
+                animate(testExecutionRow, () => {
```

Comment:
this entire block makes sense only if the row is expanded, otherwise this isn't shown and is loaded when after clicking on the row. 

As-is will lead to bugs b/c comment will be added to the list, then on expand the full comments list will be fetched again and appended to the current HTML resulting in 1 comment showing twice.

Similar issue existed for hyperlinks and is now solved by removing any HTML that may be present before appending all links, see line 321.

OTOH animate will collapse a selected row if it is expanded so an easier solution may be to just remove TE.id from the cache of expanded rows and let comment rendering/button binding happen only if the user decides to actually see that row - which may as well be never.